packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -225,7 +225,8 @@ export const VALID_PMR_LINK =
 export const VALID_PIN = /([0-9]{4,6})/;
 export const UUID_REG =
   /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
-
+export const VALID_MEETING_LINK_URL =
```

Comment:
Over here, I am not entirely comfortable with this regex, I checked the [Join Meeting API](https://developer.webex.com/docs/api/v1/meetings/join-a-meeting) and for multiple orgs it gave a large URL and this regex seemed to be the easiest way to solve it.
I tried to remove some unneccessary values and used copilot as well - however I am open to any suggestions... regex might be a bit too big.